notebooks/official/feature_store/online_feature_serving_and_fetching_bigquery_data_with_feature_store.ipynb

```diff
@@ -88,20 +88,6 @@
         "- Use the online server to fetch feature values for online prediction."
```

Comment:
<p>nit: please update the file name for this colab as "online_feature_serving_and_fetching_bigquery_data_with_feature_store_bigtable.ipynb" (i.e. add "_bigtable" suffix), to make it self-explainable comparing with optimized.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2495/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>